torch_np/_ndarray.py

```diff
@@ -395,6 +405,15 @@ def sort(self, axis=-1, kind=None, order=None):
     cumprod = _funcs.cumprod
 
     ### indexing ###
+    def item(self, *args):
+        if args == ():
+            return self.tensor.item()
+        elif len(args) == 1:
+            # int argument
+            return self.ravel()[args[0]]
+        else:
+            return self.__getitem__(args)
+
```

Comment:
Do we have tests for this?

Reply:
Not at the moment. Don't see numpy tests in numpy proper either. Behavior with arrays seems legacy TBH, so I'd prefer to put it to the priority queue.

Reply:
If you pulled this from the NumPy implementation, let's just add a link to it and roll with it.